src/Components/CartCard/CartCard.jsx

```diff
@@ -1,45 +1,61 @@
 import React from "react";
 import "./CartCard.css";
 import { Link } from "react-router-dom";
-export function CartCard() {
+import { useCart } from "../../context/cart-context";
+import { useWishlist } from "../../context/wishlist-context";
```

Comment:
Setup jsconfig file and use absolute imports

Reply:
Will make changes in next PR